libraries/standard/mqtt/utest/mqtt_lightweight_utest.c

```diff
@@ -740,6 +742,23 @@ void test_MQTT_GetUnsubscribePacketSize( void )
                                             &packetSize );
     TEST_ASSERT_EQUAL_INT( MQTTBadParameter, status );
 
+    /* Verify packet size cannot exceed limit. Note the max remaining length of
+     * an MQTT packet is 2^28-1 = 268435455, or 256MiB. Since the only way to increase
+     * the subscribe packet size is with the topic filters of the subscriptions
+     * (the lengths of which are only 2 bytes), we need at least
+     * 2^28 / 2^16 = 2^12 = 4096 of them. */
+    for( i = 0; i < 4096; i++ )
```

Comment:
Which specific line in `MQTT_GetUnsubscribePacketSize` does this provide coverage for?

Reply:
An error log that can only be reached if the packet size would exceed that allowed by the spec: https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/5ae868b852447d869b56c11f982af84a24be4993/libraries/standard/mqtt/src/mqtt_lightweight.c#L1772

Reply:
Don't we disable log messages for coverage results? Or is coverage for that branch still required when logging is disabled?

Reply:
That's possible, I know the default logging was changed to log errors for unit tests in #1096, so this will result in 100% coverage without extra cmake flags. Either way, I think this test is worth it to copy here, since it's needed in test_MQTT_GetSubscribePacketSize for non-log coverage, and it makes sense to test them both with this test.